build.gradle

```diff
@@ -21,7 +21,7 @@ plugins {
 apply plugin: 'org.springframework.boot'
 
 group 'uk.gov.hmcts.reform'
-version '0.0.16'
+version '0.0.17'
```

Comment:
need to bump the version? it's just README change?

Reply:
I know. Jenkins wants it...

Reply:
ok, then 👍